packages/documentation/src/content/docs/concepts/accounting/accounts-and-transfers.md

```diff
@@ -336,18 +349,18 @@ Rafiki transfers perform double-entry accounting. Every transfer increases both
 
 #### Payments (Same Asset)
 
-##### SPSP / Web Monetization
+##### Simple Payments Setup Protocol (SPSP) / Web Monetization (WM)
```

Comment:
Super small thing, but its Simple Payment, not Simple Payments (plural).

Reply:
Thank you @melissahenderson ! All done.